nevergrad/functions/automl/ngautosklearn.py

```diff
@@ -0,0 +1,167 @@
+                    n_jobs=1,
+                )
+                return 1 - np.mean(scores)
+            else:
+                classifier.fit(X, y)
+                return 1 - scorer(classifier, test_data[0], test_data[1])
+    except Exception:
+        return 1
+
+
+def check_configuration(config_space, values):
+    val_dict = to_dict(values[1])
+    try:
+        cs.Configuration(configuration_space=config_space, values=val_dict, allow_inactive_with_values=False)
+    except Exception:
+        return False
+    return True
+
+
+def get_config_space(X, y):
+    dataset_properties = {
+        "task": BINARY_CLASSIFICATION if len(np.unique(y)) == 2 else MULTICLASS_CLASSIFICATION,
+        "is_sparse": scipy.sparse.issparse(X),
+    }
+    return get_configuration_space(dataset_properties)
+
+
+def get_instrumention(param):
+    if isinstance(param, cs.hyperparameters.CategoricalHyperparameter):
+        return ng.p.Choice(param.choices)
+    elif isinstance(param, cs.hyperparameters.UniformIntegerHyperparameter):
+        if param.log == False:
+            return ng.p.Scalar(
+                lower=param.lower, upper=param.upper, init=param.default_value
+            ).set_integer_casting()
+        else:
+            return ng.p.Log(
+                lower=param.lower, upper=param.upper, init=param.default_value
+            ).set_integer_casting()
+    elif isinstance(param, cs.hyperparameters.UniformFloatHyperparameter):
+        if param.log == False:
+            return ng.p.Scalar(lower=param.lower, upper=param.upper, init=param.default_value)
+        else:
+            return ng.p.Log(lower=param.lower, upper=param.upper, init=param.default_value)
+    elif isinstance(param, cs.hyperparameters.Constant):
+        return ng.p.Constant(param.value)
+    raise Exception(r"{param} type not known")
+
+
+def get_parametrization(config_space: cs.ConfigurationSpace):
+    base_pipeline = [
+        "balancing:strategy",
+        "classifier:__choice__",
+        "data_preprocessing:categorical_transformer:categorical_encoding:__choice__",
+        "data_preprocessing:categorical_transformer:category_coalescence:__choice__",
+        "data_preprocessing:numerical_transformer:imputation:strategy",
+        "data_preprocessing:numerical_transformer:rescaling:__choice__",
+        "feature_preprocessor:__choice__",
+    ]
+
+    params = {}
+
+    for param in config_space.get_hyperparameters():
+        if param.name in base_pipeline:
+            if param.name in [
+                "classifier:__choice__",
+                "feature_preprocessor:__choice__",
+                "data_preprocessing:numerical_transformer:rescaling:__choice__",
+                "data_preprocessing:categorical_transformer:category_coalescence:__choice__",
+            ]:
+                params[param.name] = ng.p.Choice(
+                    [
+                        ng.p.Tuple(
+                            ng.p.Constant(param_choice),
```

Comment:
```suggestion
                            param_choice,
```
Any reason to use Constant? If param_choice is not a Parameter itself, this should be automatic